Dockerfile

```
@@ -5,8 +5,12 @@ WORKDIR /app
 
 ARG REACT_APP_GOOGLE_ANALYTICS_ID
 ENV REACT_APP_GOOGLE_ANALYTICS_ID=$REACT_APP_GOOGLE_ANALYTICS_ID
-ARG REACT_APP_SEGMENT_WRITE_KEY
-ENV REACT_APP_SEGMENT_WRITE_KEY=$REACT_APP_SEGMENT_WRITE_KEY
+ARG REACT_APP_SEGMENT_KEY
+ENV REACT_APP_SEGMENT_KEY=$REACT_APP_SEGMENT_KEY
+ARG REACT_APP_POSTHOG_KEY
+ENV REACT_APP_POSTHOG_KEY=$REACT_APP_POSTHOG_KEY
+ARG REACT_APP_GOOGLE_ANALYTICS_ID
+ENV REACT_APP_GOOGLE_ANALYTICS_ID=$REACT_APP_GOOGLE_ANALYTICS_ID
```

@ypoplavs, I recall that in Docker `ARG` is automatically set as `ENV` too - am I right and it could be simplified?

```suggestion
ARG REACT_APP_SEGMENT_KEY
ARG REACT_APP_POSTHOG_KEY
ARG REACT_APP_GOOGLE_ANALYTICS_ID
```

---

`ARG` vars are only available during the build stage and **not** in the running container, therefore, we need to have `ENV` for this.

---

Thank you, so looks like we don't need it, as we set up `ARG` here only for the environment variables that are used in the build-time.

I will keep it the same for now, but have created follow-up ticket to clean it - https://github.com/kubeshop/testkube/issues/3882